### ISSUE TITLE
Lot of lag while detecting Objects in Video file, Output is around only 2 Frame per second.

### ISSUE THREAD

**Issue by jayeshBaviskar** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#55]
I have followed the exact steps of the tutorial but the output which I am getting is only at around 2 Frame Per second. 
I am Executing the code on below-mentioned PC Configuration: 

Processor: Intel Xeon Processor - 3.60 GHz
RAM: 64 GB Ram
GPU: GeForce GTX 1060 6GB

**Comment by noelcodes** [comment]
Sound like tensorflow-gpu isn't installed properly. 
Run model again with MSI afterburner hardware monitor running in background. 
I used this to check whether the model is using my GPU, if it is, usage %  should be at least 30% or more. 
If its just hover at 0%, then you should re-install tensorflow-gpu + cuda + cudnn again. 

MSI afterburner: https://www.msi.com/page/afterburner

**Comment by karanmartian** [comment]
same issue, is there any solution. i m getting about 5 fps

**Comment by jayeshBaviskar** [comment]
No, it works well with webcam feed, but on Video it gives only around 3-5 FPS output. I think it is because the Processor uses its power to decode the video.

**Comment by karanmartian** [comment]
ok could you share your specs? i have gforce 940m 2gb, i5 processor, tf gpu. and webcam is working really slow for this, not sure why.
